_posts/2021-04-12-changelog-145.markdown

```diff
@@ -0,0 +1,89 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #145"
+date: 2021-04-12 17:00:00 +0300
+---
+
+
+## New Features
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6829 -->
+* Add `Unreachable code` inspection and quick-fix
+
+    TODO: GIF
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/7033 -->
+* Basic multiresolution at module scope in new [new name resolution](https://github.com/intellij-rust/intellij-rust/issues/6217) engine
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/7005 -->
+* Annotate wrong order of const generic parameters without `const_generics` feature
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6457 -->
+* Add inspection that annotates invalid references in `TOML` string literals. The inspection can be extended by other plugins. For example, Rust plugin provides file references to local dependency, build script and workspace paths in `Cargo.toml` as well as references to Cargo [features](https://doc.rust-lang.org/cargo/reference/features.html) and the inspection highlights all unresolved ones. Additionally, it provides a quick-fix to create a non-existent file from the corresponding file reference (by [@Kobzol])
```

Comment:
IMO this description is a bit difficult to read right now, maybe rephrase it somehow? A possible option:
```suggestion
* Add `Unresolved reference` inspection for `Cargo.toml` files. The inspection highlights unresolved references in `TOML` string literals and provides a quick-fix to create a new file for the corresponding unresolved path.
Note, the plugin provides such references for local dependencies, build scripts, workspace paths, and Cargo [features](https://doc.rust-lang.org/cargo/reference/features.html). The set of analyzed references can be extended by other plugins (by [@Kobzol])
```